dependency.lic

```diff
@@ -607,13 +616,65 @@ class ScriptManager
     end
 
     message_body += "\n****\n" * 3
-    message_body += Dir['./scripts/profiles/*.*'].join(',')
-    message_body += "\n"
-    message_body += Dir['./scripts/data/*.*'].join(',')
+    message_body += Dir['./scripts/profiles/*.*'].join(',')   # TODO: convert this to newline joins after migration of example profiles to subdir
+    message_body += "\n***\n"
+    message_body += help_me_data_checks
     message_body += "\n"
 
+    # get around pastebin's puritanical filter
+    message_body.gsub!("bastard","'child of an unwed mother(modified)'")
```

Comment:
❓ I was curious about this filter and so tried to trigger pastebin to block or put a warning up on my pastes with that word.

I was able to successfully create and view a paste as myself, as an anonymous guest, and as our Lich pastebin API key with the following with no issue. Do you of a phrase that I could test with to trip the pastebin filter?

```
curl -X POST -d 'api_dev_key=dca351a27a8af501a8d3123e29af7981' -d 'api_paste_code=a gleaming silver bastard sword with an ivory pommel' -d 'api_option=paste' "https://pastebin.com/api/api_post.php"
```

Reply:
Resolving this, we saw this today with a user, so it's certainly a thing.

```
>;e help_me('Sekmeht')
--- Lich: exec1 active.
[exec1: Bad API request, Pastebinâ€™s SMART filters have detected potentially offensive or questionable content in your paste. This paste can only be published with the visibility set to "Private".]
--- Lich: exec1 has exited.
```